admin/filters/class-stale-content-filter.php

```diff
@@ -0,0 +1,125 @@
+<?php
+/**
+ * WPSEO plugin file.
+ *
+ * @package WPSEO\Admin
+ */
+
+/**
+ * Registers the filter for filtering stale content.
+ */
+class WPSEO_Stale_Content_Filter extends WPSEO_Abstract_Post_Filter {
+
+	/**
+	 * Returns the query value this filter uses.
+	 *
+	 * @return string The query value this filter uses.
+	 */
+	public function get_query_val() {
+		return 'stale-content';
+	}
+
+	/**
+	 * Modify the query based on the seo_filter variable in $_GET
+	 *
+	 * @param string $where Query variables.
+	 *
+	 * @return string The modified query.
+	 */
+	public function filter_posts( $where ) {
+		if ( ! $this->is_filter_active() ) {
+			return $where;
+		}
+
+		global $wpdb;
+
+		$where .= sprintf( ' AND ' . $wpdb->posts . '.post_modified < "%s" ', $this->date_threshold() );
+		$where .= sprintf(
```

Comment:
The string concatenation mixed with `sprintf` feels a bit odd. Why not leverage the power of `sprintf` and use it for all the necessary variables?